### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by omarbabou**
In this PR, I implemented the following steps: 
  
        -  create a new html file called #index.html
        -  install linters
        -  implement the #Hello Microverse

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
...
+
+1. First, we must clone the repository locally in the terminal using the "git clone" command.
+2. Then create a new branch in the terminal.
+3. After creating a new branch, we will create the index.html file and then open it in the bowser.
+4. Finally do the test using the linters commands.
+
+## Authors
+
+👤 **Author1**
+
```
Comment (ShadyShawkat) [inline README.md]: - [x] Please use your name here instead of "Author1".

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (ShadyShawkat) [inline .gitignore]: - [x] Kindly add `test.md` to your `.gitignore` file, as per requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @omarbabou, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (omarbabou) [reply README.md]: okay 👍

**Review by keroloussamy — APPROVED** [review]
Hi @omarbabou, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**